### ISSUE TITLE
Images are not outputting in the listing description 

### ISSUE THREAD

**Issue by morebrownies** [OpenBazaar/openbazaar-desktop#227]
I added an image tag within the listing description field like so:

![screenshot 2016-12-16 10 46 45](https://cloud.githubusercontent.com/assets/13066886/21274421/f8dcad0e-c37c-11e6-8948-eae6460e7b0b.png)

I click **Save**, and the image is now rending inline within the edit listing modal (cool!!) 

![screenshot 2016-12-16 10 45 00](https://cloud.githubusercontent.com/assets/13066886/21274453/1db0d5ba-c37d-11e6-8967-52eee6841907.png)

I then check the listing detail screen and I see the raw tag (even after refreshing):

![screenshot 2016-12-16 10 49 14](https://cloud.githubusercontent.com/assets/13066886/21274478/4e539342-c37d-11e6-8545-760f96afa247.png)

**Comment by morebrownies** [comment]
It seems to be outputting now @jjeffryes but we probably need a max-width set on it so that it doesn't bleed out of the container. 

![screenshot 2016-12-16 12 08 15](https://cloud.githubusercontent.com/assets/13066886/21276936/9b12bb94-c388-11e6-95fe-220c33523279.png)

**Comment by rmisio** [comment]
Once our Trumbo WYSIWG editor is in, an image can be added via an interface it provides:

![image](https://cloud.githubusercontent.com/assets/794517/21321357/5197c066-c5c9-11e6-94c2-a0aeecf7a68a.png)

(the image too wide problem is still an issue that we need to fix via CSS)

This goes hand in hand with the ongoing thread in the Requirements slack channel, but do we really need to support adding raw HTML? It opens up a sanitization can of worms.

**Comment by morebrownies** [comment]
It seems there is consensus that we can skip on raw HTML support.

**Comment by jjeffryes** [comment]
#245 actually covered a size issue in the editor, which was separate, but it's not a big deal. :)

**Comment by rmisio** [comment]
> actually covered a size issue in the editor, which was separate, but it's not a big deal. :)

That was the only remaining thing left to be done in this issue.

**Comment by jjeffryes** [comment]
No, this had an issue in the listing details view where images were too big, 245 was an issue where images were too big in the editor, two separate issues. 

Not worth quibbling over. Both are fixed in #246.
